src/managers/GuildForumThreadManager.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const ThreadManager = require('./ThreadManager');
+const { TypeError } = require('../errors');
+const MessagePayload = require('../structures/MessagePayload');
+const { resolveAutoArchiveMaxLimit } = require('../util/Util');
+
+/**
+ * Manages API methods for threads in forum channels and stores their cache.
+ * @extends {ThreadManager}
+ */
+class GuildForumThreadManager extends ThreadManager {
+  /**
+   * The channel this Manager belongs to
+   * @name GuildForumThreadManager#channel
+   * @type {ForumChannel}
+   */
+
+  /**
+   * @typedef {BaseMessageOptions} GuildForumThreadCreateOptions
+   * @property {stickers} [stickers] The stickers to send with the message
+   * @property {BitFieldResolvable} [flags] The flags to send with the message
+   */
+
+  /**
+   * Options for creating a thread.
+   * @typedef {StartThreadOptions} GuildForumThreadCreateOptions
+   * @property {GuildForumThreadCreateOptions|MessagePayload} message The message associated with the thread post
+   * @property {Snowflake[]} [appliedTags] The tags to apply to the thread
+   */
+
+  /**
+   * Creates a new thread in the channel.
+   * @param {GuildForumThreadCreateOptions} [options] Options to create a new thread
+   * @returns {Promise<ThreadChannel>}
+   * @example
+   * // Create a new forum post
+   * forum.threads
+   *   .create({
+   *     name: 'Food Talk',
+   *     autoArchiveDuration: ThreadAutoArchiveDuration.OneHour,
```

Comment:
```suggestion
   *     autoArchiveDuration: 60,
```
OneHour iss for v14

Reply:
> OneHour is for v14

oki :c

Reply:
why u cant just commit suggestion?

Reply:
> why u cant just commit suggestion?

I'm using vscode, so I think it's more convenient :v

Reply:
> > why u cant just commit suggestion?
> 
> I'm using vscode, so I think it's more convenient :v

i think its just simpler to click it one time
that going to vscode and editing it